include/dpp/event_router.h

```diff
@@ -173,13 +173,13 @@ template<class T> class event_router_t {
 					from->creator->log(dpp::loglevel::ll_error, std::string{"Uncaught exception in event coroutine: "} + exception.what());
 			}
 		};
-		std::for_each(coroutine_container.begin(), coroutine_container.end(), [&](auto &ev) {
+		for (auto& ev : coroutine_container) {
```

Comment:
Did this not work with `const auto&`?

Reply:
it wasn't const before, you can't cancel it if it's const either

Reply:
It should be, that's not the actual event, but the event handler
Tbh I'd change "ev" to something more explicit here

Reply:
Added and renamed.